pkg/scm/git/git.go

```diff
@@ -34,7 +34,7 @@ type stiGit struct {
 	runner util.CommandRunner
 }
 
-var gitSshURLExp = regexp.MustCompile(`\A([\w\d\-_\.+]+@[\w\d\-_\.+]+:[\w\d\-_\.+%/]+\.git)$`)
+var gitSshURLExp = regexp.MustCompile(`\A([\w\d\-_\.+]+@[\w\d\-_\.+]+:[\w\d\-_\.+%/])`)
```

Comment:
If I add ".git" to "git@192.168.122.1:repositories/authooks" (as in "git@192.168.122.1:repositories/authooks.git"), the test case passes (with the "$" at the end). I think it needs some more investigation. Also, I'm not sure that we're doing the best validation with the current string. For example, we would allow a username that starts with "-" or "+" which I think is invalid.